backend/src/database/repositories/tenantRepository.ts

```diff
@@ -108,6 +110,7 @@ class TenantRepository {
   }
 
   static async update(id, data, options: IRepositoryOptions) {
+    TenantRepository._bustCacheForTenant(id)
```

Comment:
you need to `await` async function here - we need the cache to be busted if there is an error the update should fail otherwise we will have inconsistent data on the next fetch which will be from the cache.